src/pages/Staking/VestsList/WithdrawVest/WithdrawVest.hooks.ts

```diff
@@ -0,0 +1,48 @@
+import { BigNumber } from 'ethers';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { FOUR_WEEKS } from '../../../../constants';
+import {
+  currentBlockSelector,
+  stakingContractSelector,
+} from '../../../../store/app/app.selectors';
+import { selectedVestSelector } from '../../../../store/staking/staking.selectors';
+
+export const useGetUnlockedVesting = () => {
```

Comment:
@mwiniarsk @michalsidzej 
Guys let's discuss if we want to move this to saga that would be triggered on each block.

Reply:
I'd leave it for now, just make sure we refactor it after the app is ready

Reply:
hmm, wouldn't it be updated even on each block by useEffect -  `[selectedVestData, staking, currentBlockNumber]` when modal is active? but I agree with @michalsidzej if we plan to refactor code after some progress we can stick to this implementation. Also, we can discuss similar cases, if we want to use saga or not.

Reply:
ok, makes sense. Let's plan the refactor when all the features are ready